<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>281</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>281</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>240</width>
     <height>327</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="kernelTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>236</width>
          <height>236</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>236</width>
          <height>236</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>26</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>26</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>26</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>26</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>-1</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLineEdit" name="kernelLineEdit">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="increaseKernelSizeButton">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="decreaseKernelSizeButton">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLabel" name="kernelSizeLabel">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>      Kernel Size</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="storeKernelButton">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Save kernel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
